src/pages/carbon-offsetting.jsx

```diff
@@ -0,0 +1,564 @@
+            projects by offsetting whatever footprints remain, then eventually
+            we'll need less and less offsetting to be truly carbon neutral, then
+            shortly after be climate positive. We've got to sort this out now,
+            so if you wanna sign up for an account with an offset program today,
+            then start figuring out where to make the biggest changes in your
+            life tomorrow, that's fine. The important thing is to do both, and
+            quickly.{' '}
+          </p>
+
+          <h2 id="checkforstandards">Check for Standards</h2>
+
+          <p>
+            An offsetting organization must clearly display the independent
+            auditing or standards it uses, like{' '}
+            <a href="https://www.goldstandard.org/">Gold Standard</a>,{' '}
+            <a href="https://verra.org/">Verified Carbon Standard</a>,{' '}
+            <a href="https://www.green-e.org/programs/climate">
+              Green-e® Climate Standard
+            </a>
+            , or the United Nations{' '}
+            <a href="https://offset.climateneutralnow.org/">
+              Certified Emission Reductions
+            </a>{' '}
+            (CERs). Projects that meet the criteria for these standards are
+            rigorously audited by external auditors, usually from non-profit
+            organizations and charities like the WWF, who make sure the people
+            on the ground are indeed planting the trees, installing the solar
+            panels, etc.
+          </p>
+
+          <p>
+            Organizations like <a href="https://offset.earth/">Offset Earth</a>{' '}
+            and{' '}
+            <a href="https://co2.myclimate.org/en/offset_further_emissions">
+              myclimate
+            </a>{' '}
+            do this homework for you and only pick certified projects, so you
+            can be confident your money is getting to the best place.{' '}
+          </p>
+
+          <p>
+            If an offsetting program does not tell you which standard it's
+            using, or uses a "standard" you've never heard of, then be very
+            wary, do a load of research on that standard, or avoid it entirely.
+            We've done our due diligence on all the{' '}
+            <a href="https://protect.earth/tags/offset">offset providers</a>{' '}
+            found on our site, so maybe just stick to these ones.
+          </p>
+
+          <h2 id="sogoodorbad">So, Good or Bad?</h2>
+
+          <p>
+            If you can find an organization you trust that supports a mixture of
+            offsetting activities (not just planting trees), and do not use it
+            as an excuse to keep living a emission-heavy lifestyle, then offset
+            away.
+          </p>
+
+          <p>
+            I have stopped thinking of offsetting as supporting the status quo,
+            and think of it as a means for supporting emission reduction
```

Comment:
You wrote 'I have... and think of it...'  Fix:
'Stop thinking of... and instead think of it...'
OR
'I have...and I now think...'